__tests__/app.test.js

```diff
@@ -244,3 +244,59 @@ describe("PATCH /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: return array of comments with given article_id, sorted by earliest date first", () => {
+    return request(app)
+      .get("/api/articles/1/comment")
+      .expect(200)
+      .then(({ body }) => {
+        const commentsArray = body.comments;
+
+        expect(commentsArray).toHaveLength(11);
+
+        expect(commentsArray).toBeSortedBy("created_at", { descending: true });
+
+        commentsArray.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              article_id: expect.any(Number),
+              votes: expect.any(Number),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+
```

Comment:
We should include a test for an article that exists but there are no comments on that article - This would return a 200 as it would still be a valid request.